easybuild/easyconfigs/k/Kraken/Kraken-1.1-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PackedBinary'
+
+name = 'Kraken'
+version = '1.1'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)s/'
+description = """Kraken is a system for assigning taxonomic labels to short DNA sequences,
+ usually obtained through metagenomic studies. Previous attempts by other
+ bioinformatics software to accomplish this task have often used sequence
+ alignment or machine learning techniques that were quite slow, leading to
+ the development of less sensitive but much faster abundance estimation
+ programs. Kraken aims to achieve high sensitivity and high speed by
+ utilizing exact alignments of k-mers and a novel classification algorithm."""
+
+# Part is compiled with CXX, the rest is in Perl
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'openmp': True}
+
+github_account = 'DerrickWood'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_CXX-CXXFLAGS.patch']
+checksums = [
+    'a4ac74c54c10920f431741c80d8a172670be12c3b352912000030fb5ea4c87a7',  # v1.1.tar.gz
+    '84c017d6a80ccaac1e23561c83cac90bcc3d62baa8d31657a7dafbc2c3f29726',  # Kraken-1.1_CXX-CXXFLAGS.patch
+]
+
+dependencies = [
+    ('Perl', '5.28.0'),
+    ('Jellyfish', '1.1.12'),
```

Comment:
@SethosII Any reason for not using the existing `Jellyfish-2.2.10-foss-2018b.eb` here?

The existing `Kraken-1.0-intel-2018a-Perl-5.26.1.eb` also uses `Jellyfish` 2.x

Reply:
The documentation states that only `Jellsfish` 1 is supported:

> Finally, if you want to build your own database, you will need to install the Jellyfish $k$-mer counter. Note that Kraken only supports use of Jellyfish version 1. Jellyfish version 2 is not yet compatible with Kraken.
>
> -- https://github.com/DerrickWood/kraken/blob/master/docs/MANUAL.markdown

Therefore I didn't try with `Jellyfish` 2.

Reply:
OK, we should clarify this with a comment above this dep, to prevent us from accidentally jumping to `Jellyfish` 2.x.

And we should fix `Kraken-1.0-intel-2018a-Perl-5.26.1.eb`?

Reply:
Fixed `Jellyfish` dependency for existing `Kraken` 1.0 easyconfig in #7743 

@SethosII Can you please add a clarifying comment above this line like I did in https://github.com/easybuilders/easybuild-easyconfigs/pull/7743/files#diff-70e7520a466d3046e0604a776d5794c4R30?

Reply:
@boegel I added a comment.